debian/tracker/debian.go

```diff
@@ -351,3 +370,15 @@ func (c Client) parseSources(sourcePath string) ([]textproto.MIMEHeader, error)
 
 	return headers, nil
 }
+
+// ref. https://salsa.debian.org/security-tracker-team/security-tracker/-/blob/50ca55fb66ec7592f9bc1053a11dbf0bd50ee425/lib/python/bugs.py#L402
+func tempBugName(bugNumber int, description string) string {
+	switch {
+	case strings.HasPrefix(description, "["):
+		description = strings.TrimPrefix(strings.TrimSuffix(description, "]"), "[")
+	case strings.HasPrefix(description, "("):
+		description = strings.TrimPrefix(strings.TrimSuffix(description, ")"), "(")
+	}
```

Comment:
Are there cases when description only contains suffix `]` (`)`) and doesn't contain prefix `[`(`(`)?
if not - it looks like we can remove `switch`.

Reply:
First, when calculating the hash of description with md5, it seems necessary to remove the parentheses at both ends.

Referring to the implementation below, if the first character of the description has either `(` or `[`, an error will occur if the last character is not closed with the corresponding parentheses such as `)` or `]`. 
https://salsa.debian.org/security-tracker-team/security-tracker/-/blob/778d44678936649eb550e589645c60a50da81f88/lib/python/sectracker/parsers.py#L308-313

In other words, the case you mentioned does not seem to exist.

Reply:
For example, if the description is `[description()]`, if you randomly delete `[(` from the prefix and `)]` from the suffix, it will become `description(`).
In this case, we expect it to be `description()`.
For this purpose, I adopted an implementation that looks at the type of parenthesis in the first character and determines the type of parenthesis to be removed from the prefix and suffix.

Reply:
I suspected there might be something like this.
Thanks for example.